portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;i
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+		
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+	@Override
+	public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Try to avoid having lines longer than 80 characters (for style purposes and to make it easier to read). See here for one resource (item #9): https://www.cs.swarthmore.edu/~newhall/unixhelp/javacodestyle.html

Instead, IOException should be on a new line and properly indented

Reply:
Made formatting changes